pom.xml

```diff
@@ -99,6 +99,8 @@
         <dependency>
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
+            <version>2.7.1</version>
```

Comment:
Just like in Enterprise, can we update ModuleParent and use version from parent?